.github/workflows/test.yml

```diff
@@ -39,3 +39,24 @@ jobs:
     - name: Run Test
       run: |
         `which django-admin` test django_rq --settings=django_rq.tests.settings --pythonpath=.
+
+  mypy:
+    runs-on: ubuntu-latest
+    name: Type check
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v4.2.0
+      with:
+        python-version: "3.8"
```

Comment:
Picking an old supported Python version since it doesn't it's tested otherwise, and this will make sure the type annotations don't break the older Pythons